app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+    validates :title, presence: true
+    belongs_to :user
```

Comment:
- [ ] There is an association missing here, related with comments. Think about it :)  

_As it was said in the odin project, post should be able to do [this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/post_has_many_comments.png)_